hypothesis-python/tests/array_api/test_arrays.py

```diff
@@ -18,11 +18,23 @@
 from hypothesis import assume, given, strategies as st
 from hypothesis.errors import InvalidArgument
 from hypothesis.extra.array_api import DTYPE_NAMES, NUMERIC_NAMES
+from hypothesis.internal.floats import width_smallest_normals
 
-from tests.array_api.common import COMPLIANT_XP, xp, xps
+from tests.array_api.common import COMPLIANT_XP, WIDTHS_FTZ, xp, xps
 from tests.common.debug import find_any, minimal
 from tests.common.utils import fails_with, flaky
 
+pytestmark = [
+    pytest.mark.skipif(
+        not isinstance(-1.1754944e-38, float),
+        reason=(
+            "Most likely using a 32-bit build Python, "
+            "which might not represent 64-bit subnormals as floats, "
+            "failing sanity check in internal.floats.next_up"
```

Comment:
I think we tracked this down to `np.float32` instead?

Reply:
Yep forgot to remove this. I think just removing it is fine now, will need to see win CI.